kuma/users/jinja2/users/includes/stripe_subscription.html

```diff
@@ -0,0 +1,71 @@
+<noscript>
+  <style>
+    .stripe-subscription {
+      display: none
+    }
+  </style>
+</noscript>
+
+<section class="stripe-subscription">
+  {% stylesheet 'stripe-subscription' %}
+
+  {% if not subscription_info %}
+    <form id="stripe-form" action="{{ url('users.create_stripe_subscription') }}" method="post">
+      {% csrf_token %}
+      <h2>{{ _('Help grow MDN') }}</h2>
+      {% if has_stripe_error %}
+          <div class="stripe-error">
+              <h3>{{ _('Stripe error') }}</h3>
+              <p>
+                  {{ _("An error occurred trying to set up the subscription with Stripe's server. We've recorded the error and will investigate it.") }}
+              </p>
+              <hr>
+          </div>
+      {% endif %}
+      <p>
+        {{ _("Be a part of making MDN even better. You'll get back more of the knowledge and tools you rely on for when your work has to work.") }}
```

Comment:
@schalkneethling Should we use a special apostrophe inside English strings or is it OK with a regular single quote?
Also, if so, do you know why?